cypress/e2e/functional/opal/manualAccountCreation/adultOrYouthOnly/PO-358_contactDetails.feature

```diff
@@ -18,112 +18,130 @@ Feature: PO-358_Contact Details for adult or youth only
 
     Then I see "Create account" as the caption on the page
     Then I see "Account details" on the page header
+    Then I see the status of "Contact details" to "Not provided"
 
     Then I click on "Contact details" link
     Then I see "Contact details" on the page header
 
-  Scenario Outline: AC1, AC3, AC4- positive: verifying all the fields with maximum mo.of characters
+  Scenario Outline: AC1(419 & 358), AC3, AC4(419 & 358)AC5(419)- positive: verifying all the fields with maximum mo.of characters in all fields and spaces in telephone numbers
     When I enter primary email address "<primaryEmail>"
     When I enter secondary email address "<secondaryEmail>"
     When I enter mobile telephone number "<mobileTelephone>"
     When I enter home telephone number "<homeTelephone>"
-    When I enter business telephone number "<businessTelephone>"
+    When I enter work telephone number "<workTelephone>"
```

Comment:
The generic steps can handle these inputs

Reply:
How should non-business peple understand what is the field name if I haven't say in feature file

Reply:
'When I enter "<workTelephone>" into the "Work telephone number" field' can be used indicating the value entered and the field name. Meaning new steps don't have to be created

Reply:
Non-business/non-technical people will easily understand if we say about field names

Reply:
The generic step specifies the field name as a string so anyone reading the feature file will understand which field is being interacted with.